functions.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * display all products in collection
+ * @param $products
+ * @return string
+ */
+function displayAllProducts(array $products): string {
+    $result = '';
+    foreach ($products as $product) {
+        $result .= displayProduct($product);
+    }
+    return $result;
+}
+
+/**
+ * @param $db
+ * @return array
+ */
+function getProducts($db): array
+{
+    $query = $db->prepare("SELECT `name`, `year-released`, `color-way`, `size`, `price`, `images` FROM `sneakers`;");
+    $query->execute();
+    return $query->fetchAll();
+}
+
+/**
+ * split products into individual items
+ * @param $product
+ * @return string
+ */
+function displayProduct(array $product): string
+{
+    $result = '<div class="product"><img src="' . $product['images'] . '" alt="nikeshoe" width=550 height=360; />';
```

Comment:
I don't think your css uses the product class.  If Charlie is going to review it next, might be worth removing.  However, if you're planning on adding more css that utilises the product leave it in for a while.